src/main/java/com/checkout/common/MarketplaceDataSubEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.checkout.common;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public final class MarketplaceDataSubEntity {
+
+    private String id;
+
+    private Long amount;
+
+    private String reference;
+
+    @SerializedName("commission")
```

Comment:
Why does this serialization is needed? (Just curious)

Reply:
No need, will remove it.